cmd/info.go

```diff
@@ -14,11 +17,19 @@ var infoCmd = &cobra.Command{
 	Use:   "info",
 	Short: "Print the basic information of xud-launcher and your xud-docker setups",
 	Run: func(cmd *cobra.Command, args []string) {
-		// TODO implement the info command
-		// return
-		// * xud-launcher version
-		// * xud-docker network dir location
-		// * xud-docker installed or not
-		fmt.Println("To be implemented!")
+		fmt.Printf("Launcher Version: %s\n", GetVersion())
+		fmt.Printf("Built At: %s\n", GetBuildTime())
+		fmt.Printf("XUD-Docker Version: 20.12.07\n")
```

Comment:
This is still hardcoded. But that's fine I guess for this release

Reply:
@michael1011 Yep. Let's leave this to following PRs.